recipes/wasmtime/all/conanfile.py

```diff
@@ -0,0 +1,80 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration, ConanException
+import os
+
+
+class WasmtimeConan(ConanFile):
+    name = 'wasmtime'
+    homepage = 'https://github.com/bytecodealliance/wasmtime'
+    license = 'Apache License 2.0'
+    url = 'https://github.com/conan-io/conan-center-index'
+    description = "Standalone JIT-style runtime for WebAssembly, using Cranelift"
+    topics = ("webassembly", "wasm", "wasi")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        'fPIC': [True],
+    }
+    default_options = {
+        'shared': False,
+        'fPIC': True,
+    }
+    generators = "cmake", "cmake_find_package", "cmake_find_package_multi"
```

Comment:
No build

```suggestion
    no_copy_source = True
```

Reply:
How `conan` will generate conanfile.cmake with all constants ?

Reply:
That file is not used by the recipe 😕 I am not sure what you have misunderstood but if you can explain more I'd love to help sort that out together

Reply:
I need conanfile.cmake to be generated for consuming project ...
How in such case conanfile.cmake will be created ?

Reply:
Ahhhhhh this setting is per recipe. This one does not impact consumers, they need to add it to their own conan files to have a unique individual experience!

Hopefully that make sense.

This recipe does not use those hence why it can be removed

Reply:
Done